erreser.go

```
@@ -507,7 +891,18 @@ func (c *addFeedCmd) Execute(_ context.Context, f *flag.FlagSet, _ ...interface{
 		return subcommands.ExitUsageError
 	}
 
-	addFeed(c.feed)
+	// Need to initialize DB here for the command-line tool
+	dbParams := database.InitDbParams()
+	db := database.DbConnect(dbParams)
+	db.AutoMigrate(&models.Feed{}, &models.Image{}) // Ensure tables are created
```

golangci-lint (1.64.8)

Error return value of `db.AutoMigrate` is not checked

(errcheck)